src/map/lua/lua_zone.cpp

```diff
@@ -200,7 +200,15 @@ bool CLuaZone::isNavigablePoint(const sol::table& point)
     };
     // clang-format on
 
-    return m_pLuaZone->m_navMesh->validPosition(position);
+    if (m_pLuaZone->m_navMesh)
+    {
+        return m_pLuaZone->m_navMesh->validPosition(position);
+    }
+    else
+    {
+        ShowError("CLuaZone::isNavigablePoint(): m_pLuaZone->m_navMesh is nullptr! Is your navmesh git submodule up to date?");
```

Comment:
Those of us not using a navmesh don't want our logs spammed with this every time the function is called man.

Reply:
It appears I did not understand it was supposed to be optional, my bad!